Apps/UnitTests/Shared/Tests.cpp

```diff
@@ -15,10 +14,10 @@
 std::unique_ptr<Babylon::AppRuntime> runtime{};
 std::unique_ptr<Babylon::Graphics> graphics{};
 
-int main()
+int run()
 {
-    Babylon::ContextConfiguration graphicsConfig{};
-    graphics = Babylon::Graphics::CreateGraphics(graphicsConfig);
+   Babylon::ContextConfiguration graphicsConfig{};
+   graphics = Babylon::Graphics::CreateGraphics(graphicsConfig);
```

Comment:
weird spacing changes?

Reply:
Not sure why this looks weird on GitHub - if you open the file, the spacing seems okay, unless I am missing something